internal/indexer/indexer.go

```diff
@@ -642,21 +730,23 @@ func (i *Indexer) indexReferenceToExternalDefinition(p *packages.Package, docume
 	})
 
 	rangeID, _ := i.ensureRangeFor(pos, definitionObj)
-	refResultID := i.emitter.EmitReferenceResult()
-	_ = i.emitter.EmitTextDocumentReferences(rangeID, refResultID)
-	_ = i.emitter.EmitItemOfReferences(refResultID, []uint64{rangeID}, document.DocumentID)
-
 	if hoverResultID != 0 {
 		_ = i.emitter.EmitTextDocumentHover(rangeID, hoverResultID)
 	}
 
+	// You might think, "Hey, TJ, don't you want to emit a textDocument/references result here?"
+	//   Great question!
+	//   No.
```

Comment:
Why not?

Reply:
If we make a reference here, we won't look up the import moniker, which is what we really want. That will prevent linking these correctly between packages.

Reply:
ok - can you put this in the comment? 🙂